mk/travis.sh

```diff
@@ -21,30 +21,22 @@ printenv
 
 case $TARGET_X in
 aarch64-unknown-linux-gnu)
-  DL_TARGET=aarch64-linux-gnu
-  DL_DIGEST=b9137008744d9009877f662dbac7481d673cdcb1798e727e325a37c98a0f63da
+  sudo apt-get install --no-install-recommends g++-aarch64-linux-gnu gcc-aarch64-linux-gnu libc6-dev-arm64-cross -y
```

Comment:
Do we need to use `sudo apt-get install` to install these, or can we add them as package dependencies the normal way in update_travis_yml.py?

If these packages can only be installed using `sudo` then please add a note to that effect so people can understand why we're doing this a different way. Otherwise, let's switch to the declarative way.